doc/user_guide/user_guide.md

```diff
@@ -0,0 +1,103 @@
+# error-reporting-lua &mdash; User Guide
+
+This project contains the source code for the `exaerror` Lua module. This modules lets you define errors with a uniform set of attributes. The created error objects can be used in places where strings are expected like in string concatenation.
+
+And you can conveniently rais a Lua `error` from them.
+
+Additionally the resulting code is made to be parseable, so that you can extract an error catalog from the code.
+
+# Defining Error Objects
+
+The core concept of the module is an error object defined in `exaerror`. This is a Lua table that has a couple of predefined attributes:
+
+* `code`: A machine readable error code
```

Comment:
On the confluence page you called something like `E-TEST-1` error-identifier.
In the other projects we try to use this terminilogy.

Reply:
Change explanation term to "identifier".